psa-crypto-sys/build.rs

```diff
@@ -209,22 +276,17 @@ mod common {
 #[cfg(all(feature = "interface", not(feature = "operations")))]
 mod interface {
     use super::common;
-    use std::env;
-    use std::io::{Error, ErrorKind, Result};
+    use std::io::Result;
 
     // Build script when the interface feature is on and not the operations one
     pub fn script_interface() -> Result<()> {
-        if let Ok(include_dir) = env::var("MBEDTLS_INCLUDE_DIR") {
-            common::configure_mbed_crypto()?;
-            common::generate_mbed_crypto_bindings(include_dir.clone(), false)?;
-            let _ = common::compile_shim_library(include_dir, true, false)?;
-            Ok(())
-        } else {
-            Err(Error::new(
-                ErrorKind::Other,
-                "interface feature necessitates MBEDTLS_INCLUDE_DIR environment variable",
-            ))
-        }
+        let include_dir = common::get_external_mbedtls_include_only().unwrap();
```

Comment:
We are not unwrapping in the rest of the function calls, should we not use `?` and handle the error afterwards as well?

Reply:
> We are not unwrapping in the rest of the function calls, should we not use `?` and handle the error afterwards as well?

That's not true, on current main their are 19 uses of unwrap():
  - `git grep unwrap build.rs | wc -l`

I mean we can use question mark operator here I'm not opposed, but it didn't seem like a recoverable error so panic would suffice and the output is pretty identical. But ill switch to `?`

With panic on unwrap:
```
  --- stderr
  thread 'main' panicked at psa-crypto-sys/build.rs:283:71:
  called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "interface feature necessitates MBEDTLS_INCLUDE_DIR environment variable" }
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
 ```
 
 Versus `?`:
 ```
  --- stderr
  Error: Custom { kind: Other, error: "interface feature necessitates MBEDTLS_INCLUDE_DIR environment variable" }
```

Reply:
Sorry, what I meant to say is "in the rest of the function call", my bad.